setup.py

```diff
@@ -16,9 +16,11 @@
 
 f = open("./requirements.txt",'r')
 install_req=f.readlines()
-# install_req = [req.replace('\n', '').strip() for req in install_req]
 f.close()
 
+
+install_req += ['oda_api', 'dispatcher-plugin-integral-all-sky']
```

Comment:
if anything, `install_req` is meant to be more narrow than `requirements.txt`

see [there](https://packaging.python.org/discussions/install-requires-vs-requirements/)
```
Whereas install_requires requirements are minimal, requirements files often contain an exhaustive listing of pinned versions for the purpose of achieving repeatable installations of a complete environment.
```